app_dart/app.yaml

```diff
@@ -8,3 +8,12 @@ service: default
 
 resources:
   memory_gb: 4.0
+
+handlers:
+# The Dart server handles all requests. However, this is used to ensure the 
+# assets the dart server needs are uploaded to AppEngine.
+- url: /v2/(.*\.(html|css|js|ico|svg|png|jpg|map))$
+  application_readable: true # So the dart server can read the files
+  static_files: build/web/\1
```

Comment:
If there's no built-in static file handling for custom runtimes in GAE, then are the `static_files` and `secure` lines needed?

Reply:
Removed `secure`.

`static_files` cannot be removed as handler requires some sort of url handling logic. I simplified so that it just returns `index.html` just in case this changes in the future. That way we know about the change and can remove the dart handler code.